requirements/requirements-common.txt

```diff
@@ -6,4 +6,5 @@ pytz==2016.4
 python-social-auth==0.2.19
 selenium==2.53.6
 coverage
-ruamel.yaml==0.15.23
\ No newline at end of file
+ruamel.yaml==0.15.23
```

Comment:
Why do we have hard requirements?  Why not `> 0.15` or so for all of these?

Reply:
Specific pinning is usually recommended, so as to avoid bugs/errors arising out of incompatible dependencies.